utils/wasm-builder/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "substrate-wasm-builder"
-version = "3.0.0"
+version = "4.0.0"
```

Comment:
You will probably want to revert this, as the wasm-builder was already on `3.0.0`. This also lead to the release of wasm-builder `4.0.0` on [crates.io](https://crates.io/crates/substrate-wasm-builder) which is equivalent to `3.0.0`.

Reply:
We can not revert this, it is now out. However, you are right that this never have happened...